tests/3d_examples/test_3d_load_image/load_image.cpp

```diff
@@ -0,0 +1,84 @@
+ /**
+ * @file 	load_image.cpp
+ * @brief 	This is the test of using distance map to generate body fitted particles (3D).
+ * @details We use this case to test the particle generation and relaxation for a complex geometry. 
+ *			Before particle generation, we clean the sharp corners of the model. 
+ * @author 	Yongchuan Yu and Xiangyu Hu
+ */
+
+#include "sphinxsys.h"
+
+/** case file to setup the test case */
+#include "case.h"
+
+using namespace SPH;
+
+int main(int ac, char* av[])
+{
+	//----------------------------------------------------------------------
+	//	Build up -- a SPHSystem
+	//----------------------------------------------------------------------
+	SPHSystem system(system_domain_bounds, dp_0);
+	/** Tag for run particle relaxation for the initial body fitted distribution. */
+	system.run_particle_relaxation_ = true;
+	//handle command line arguments
+	#ifdef BOOST_AVAILABLE
+	system.handleCommandlineOptions(ac, av);
+	#endif
+	/** output environment. */
+	In_Output 	in_output(system);
+	//----------------------------------------------------------------------
+	//	Creating body, materials and particles.
+	//----------------------------------------------------------------------
+	ImportedModel* imported_model = new ImportedModel(system, "ImportedModel");
+	SolidParticles imported_model_particles(imported_model);
+	imported_model_particles.addAVariableToWrite<indexScalar, Real>("SmoothingLengthRatio");
+	//----------------------------------------------------------------------
+	//	Define simple file input and outputs functions.
+	//----------------------------------------------------------------------
+	BodyStatesRecordingToVtu		write_imported_model_to_vtu(in_output, { imported_model });
+	MeshRecordingToPlt 	cell_linked_list_recording(in_output, imported_model, imported_model->cell_linked_list_);
+	//----------------------------------------------------------------------
+	//	Define body relation map.
+	//	The contact map gives the topological connections between the bodies.
+	//	Basically the the range of bodies to build neighbor particle lists.
+	//----------------------------------------------------------------------
+	BaseBodyRelationInner* imported_model_inner
+		= new BodyRelationInnerVariableSmoothingLength(imported_model);
```

Comment:
it might not matter in this main function but "new" keyword is not used properly. please consisder using smart pointer